scripts/globals/items.lua

```diff
@@ -759,12 +771,19 @@ xi.items =
     IMPERIAL_MYTHRIL_PIECE          = 2186,
     IMPERIAL_GOLD_PIECE             = 2187,
     FIENDISH_SKIN                   = 2189,
+    APPRAISAL_SWORD                 = 2190,
     UNAPPRAISED_SWORD               = 2190,
+    APPRAISAL_DAGGER                = 2191,
     UNAPPRAISED_DAGGER              = 2191,
+    APPRAISAL_POLEARM               = 2192,
```

Comment:
duuuuuuuuuuuuuuuuplicates

Reply:
These APPRAISAL_x items still need to be removed.  UNAPPRAISED_x is the format we're using here, and already exist

Reply:
> These APPRAISAL_x items still need to be removed. UNAPPRAISED_x is the format we're using here, and already exist

understood, these have been removed.